### ISSUE TITLE
Error setting a custom asset with certificate.

### ISSUE THREAD

**Issue by avelad** [google/shaka-player#1914]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?** Yes

**What version of Shaka Player are you using?** Master

**Can you reproduce the issue with our latest release version?** N/A

**Can you reproduce the issue with the latest code from `master`?** Yes

**Are you using the demo app or your own custom app?** Demo App

**If custom app, can you reproduce the issue using our demo app?**

**What browser and OS are you using?**
Safari 12.1 Mojave

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**

**What are the manifest and license server URIs?**

Any with a certificate url (with a fake custom asset the issue is reproducible)

**What did you do?**

Create a custom asset with certificate uri

**What did you expect to happen?**
No errors

**What actually happened?**

Uncaught (in promise) TypeError: Cannot set property 'serverCertificate' of undefined
    at ShakaDemoMain.drmConfiguration_ (main.js:693)
    at async ShakaDemoMain.loadAsset (main.js:725)

I think that the issue is in https://github.com/google/shaka-player/blob/master/demo/main.js#L693

**Comment by theodab** [comment]
Thanks for the report! License server certificate URI is, admittedly, a feature I haven't tested very much, so it's likely I introduced some bug into it during a refactor or something. I'll look into it.

**Comment by theodab** [comment]
I fixed the bug you found.
I'm not entirely sure whether or not the actual certificate URI field works as intended; every bit of sample content I've found to test it has had some kind of unrelated issue or another (CORS errors, missing resources, etc).
So I'm going to close this issue. If you have some private content of your own which can verify whether or not the field works, we'd be happy to fix any bugs you find.
Thanks for the bug reports!
